guides/common/assembly_converting-a-host-with-convert2rhel.adoc

```diff
@@ -1,56 +1,10 @@
 ifdef::context[:parent-context: {context}]
 
-[id="assembly_converting-a_host-with-convert2rhel_{context}"]
-= Converting a Host to {RHEL} with Convert2RHEL
-
-Convert2RHEL enables the conversion of {RHEL} derivative distributions into a supportable {RHEL} state while retaining existing applications and configurations.
-
-The conversion process is similar to a minor release upgrade of {RHEL} in which every RPM package on the system is replaced.
-Third-party packages and non-{Team} packages not available in {RHEL} are not replaced.
-
-Convert2RHEL removes unnecessary packages such as logos, or packages known to cause issues during the conversion, and replaces the `CentOS-release` package with the `rhel-release` package.
-It subscribes the system to {ProjectName} and replaces all packages signed by CentOS.
-ifdef::satellite[]
-For {Project}, it also subscribes the system to {Team} Subscription Management.
-endif::[]
-
-The amount of time taken to do this can vary depending on how many packages need to be replaced, network speed, storage speed, and similar variables.
-
-Use the global registration template to register and subscribe your system before the conversion.
-For more information, see {ManagingHostsDocURL}registering-a-host_managing-hosts[Registering a Host to {ProjectName} Using the Global Registration Template] in the _Managing Hosts_ guide.
-
-To convert hosts to {RHEL} with convert2RHEL, you must complete the following procedures:
-
-. xref:installing-convert2rhel_{context}[].
-. xref:Creating_an_Activation_Key_{context}[].
-. xref:preparing-the-host-group_{context}[].
-. xref:registering-hosts-for-conversion_{context}[].
-. xref:converting-hosts-with-convert2rhel_{context}[]
-
```

Comment:
I'd keep the title and intro in the assembly and put only the Prerequisites into a separate concept module.